src/pages/publication/references_tab/JournalPublicationReferenceForm.tsx

```diff
@@ -108,7 +109,7 @@ const JournalPublicationReferenceForm: React.FC = () => {
       <StyledPeerReview>
         <PeerReview fieldName={JournalPublicationFieldNames.PEER_REVIEW} label={t('references.peer_review')} />
       </StyledPeerReview>
-      <NviValidation isPeerReviewed={isPeerReviewed} isRated={!!isRatedJournal} />
+      <NviValidation isPeerReviewed={isPeerReviewed} isRated={!!isRatedJournal} dataTestId="nvi_journal" />
```

Comment:
Bare et kommentar-spørsmål :) Har vi tenkt på en navngiving-standard for testid-er? Disse er vel globale for hele applikasjonen? Får vi veldig mange kan vi risikere konflikter.

Reply:
ja, vi burde tenke på det. nå er det veldig mange forskjellige format; både camelCase og snake. kan ta en beslutning på dette når Eirik er tilbake kanskje? :)

Reply:
Neee. bare ta en sjefsavgjørelse du :-)